.github/workflows/ci.yaml

```diff
@@ -96,11 +96,11 @@ jobs:
         kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/release-0.42/example/prometheus-operator-crd/monitoring.coreos.com_prometheusrules.yaml
         kubectl apply -f ci/prom-rule-jsonnet.yaml
 
-        timeout 30s bash -c 'until kubectl get cm grafana-dashboards-generated &> /dev/null; do sleep 2; done'
+        timeout 30s bash -c 'until kubectl get cm grafana-dashboards-generated &> /dev/null; do sleep 2; done' || echo "Failed configmaps grafana-dashboards-generated not found"
```

Comment:
I would add ```>&2``` after the echo to redirect the message to stderr.
Add ```; false``` at the end of line, so the whole statement still returns nonzero value when the error message is printed.
So the end of line will look like this:
```
 || echo "Failed configmaps grafana-dashboards-generated not found" >&2; false
```
Same for line 103.

Reply:
My mistake, we have to use parentheses to override operator prededence, otherwise false is always executed:
```
|| (echo "Failed configmaps grafana-dashboards-generated not found" >&2; false)
```

But now taht we use 'set -x' this part of statement (the one after ||) is not important anymore. You can remove it if you like.